src/Tools/dotnet-monitor/Commands/GenerateApiKeyCommandHandler.cs

```diff
@@ -55,6 +55,8 @@ public static Task<int> Invoke(CancellationToken token, OutputFormat output, ICo
             }
             else
             {
+                outputBldr.AppendLine(ExperienceSurvey.ExperienceSurveyMessage);
```

Comment:
Are we sure we want to add this for GenerateKey? For example, this breaks my setup scripts since I haven't yet switched to machinejson.

Reply:
All of the other non`machinejson` output formats have a lot of human  text already so I don't see this as changing much. I assume your script does some kind of beeline splitting and assuming an index if you haven't moved to `machinejson` yet.